src/app/components/side-panel/side-panel.component.html

```diff
@@ -1,9 +1,9 @@
-<a [routerLink]="[]" [queryParams]="{}">
+<div [routerLink]="[]" [queryParams]="{}">
```

Comment:
It has to be an anchor tag else you will not be able to right click on it and open it in a new tab.